tools/authoring-decisions.adoc

```diff
@@ -3,10 +3,437 @@ include::../partials/attributes.adoc[]
+
+.DRD alteranative representation for the Input Data node
+image::tools/authoringDecisions/new-editor-diagram-drd-overview-alternative-input.png[width=500px]
+
+Also, it's possible to change the classic shape to an alternative one, making it more visually cohesive with the BPMN model. This change affects the entire DRD, so it's not possible to have just one node with an alternative shape.
+
+==== 4.Properties panel
+
+The properties panel lets you manage the properties of a Decision element. Enabling the possibility to add detailed descriptions, and documentation links and tweak the font and shape styles.
+
+==== 5.Overlay properties
+
+On the Overlay properties, it's possible to tweak user preferences, which helps with visualization and organization.
+
+. Snapping -> Makes the elements snap into an invisible grid. It's possible to tweak the grid layout by changing the "Horizontal" and "Vertical" values.
+
+.Enanle/disable snapping
+image::tools/authoringDecisions/new-editor-overlay-snapping.gif[]
+
+. Highlight selected node(s) hierarchy -> This option is very useful for complex models where the user needs to verify which nodes are somehow impacting the selected node.
+
+.Enanle/disable node hierarchy
+image::tools/authoringDecisions/new-editor-overlay-hierarchy.gif[]
+
+. Show data type toolbar on nodes -> Control if the data type toolbar is visible or not on the Input Data, Decision, BKM, and Decision Service nodes.
+
+.Enable/disable data type toolbar on nodes
+image::tools/authoringDecisions/new-editor-overlay-show-data-type.gif[]
+
+. Enable styles -> It's possible to turn on/off the node styles. It's important to notice that this option will not change the file, only the visual representation.
+
+.Enable/disable data type toolbar on nodes
+image::tools/authoringDecisions/new-editor-overlay-style.gif[]
+
+==== 6.Autolayout (beta)
+
+The Autolayout is still a beta feature. It will automatically organize your model placing the nodes in a way they're close to the ones they have a relationship.
+
+.Enable/disable data type toolbar on nodes
+image::tools/authoringDecisions/new-editor-autolayout.gif[]
+
+==== 7.Visualization controls
+
+The visualization controls give the user the option to zoom in/out from the model, to make the viewport fit the current model, or to lock any interaction.
+
+=== B. Data types
+
+The data types tab is where all custom data types of the DMN model are placed. The New DMN Editor brings a brand new experience for managing custom data types. The following two images show a model without custom data types and the next are the data types of the DMN sample.
+
+.Empty data types tab
+image::tools/authoringDecisions/new-editor-data-types-empty.png[]
+
+.Sample data types tab
+image::tools/authoringDecisions/new-editor-data-types-sample.png[]
+
+==== Adding a data type
```

Comment:
I'm wondering if this whole section should be a table or arranged some other way to help put the images closer to the text (left or right).

Reply:
I'm not sure if this would work as some images are big.